config-helper.js

```diff
@@ -50,6 +50,33 @@ module.exports = {
 			this.replaceMain({entry: config.entry[opts.chunk || 'main']}, replacement, opts);
 		}
 	},
+	replaceEntry: function (config, replacement, opts = {}) {
+		if (typeof replacement === 'string') {
+			try {
+				replacement = JSON.parse(replacement);
+			} catch (e) {
+				replacement = JSON.parse('{"main": "' + replacement + '"}');
+			}
+		}
+		const {main, ...restEntries} = replacement;
+
+		if (main !== undefined) this.replaceMain(config, main, opts);
+
+		if (Object.keys(restEntries).length !== 0) {
+			config.entry = {...config.entry, ...restEntries};
+			if (config.optimization.splitChunks?.chunks !== undefined) {
+				config.optimization.splitChunks.chunks = 'all';
+			} else {
+				config.optimization.splitChunks = {...config.optimization.splitChunks, chunks: 'all'};
+			}
```

Comment:
Is there anywhere using `chunks` of splitChunks?
If not, we could do just
```suggestion
			config.optimization.splitChunks.chunks = 'all';
```

Reply:
This statement needed, so should be remained

Reply:
Please be more specific. Your answer is not convincing without any clue or example.